aa_stripe/models.py

```diff
@@ -125,6 +125,52 @@ def create_at_stripe(self):
         self.save()
         return self
 
+    def update_at_stripe(self, stripe_token, set_default):
```

Comment:
I am lost in this function. Please add some comments.

Reply:
I'm not sure what is not clear.